src/p2p/wire/messages.ts

```diff
@@ -0,0 +1,79 @@
+
+export interface Request {
+  id: number;
+  method_id: number;
+  body: string;
```

Comment:
To make it more clearer, you should probably create empty interface RequestBody and each message should extend it so you can put RequestBody as body type